pints/__init__.py

```diff
@@ -205,6 +205,7 @@ def version(formatted=False):
 from ._mcmc._metropolis import MetropolisRandomWalkMCMC
 from ._mcmc._slice_stepout import SliceStepoutMCMC
 from ._mcmc._slice_doubling import SliceDoublingMCMC
+from ._mcmc._slice_hyperrectangles import SliceHyperrectanglesMCMC
```

Comment:
Can we rearrange to make alphabetical (also switching stepout and doubling order)?